Tryouts/Messaging/Client/MessageRouterClient.cs

```diff
@@ -70,10 +82,11 @@ public ValueTask ConnectAsync(CancellationToken cancellationToken = default)
         string? payload = null,
         CancellationToken cancellationToken = default)
     {
-        var requestId = Guid.NewGuid().ToString();
+        var requestId = new string(Guid.NewGuid().ToString());
```

Comment:
Please double check if new string allocation is really needed here.

Reply:
That's definitely not right, I'll fix it.